src/haddock/libs/libalign.py

```diff
@@ -1360,3 +1360,46 @@ def check_common_atoms(models, filter_resdic, allatoms, atom_similarity):
             )
         raise ALIGNError(_err_msg)
     return n_atoms, list(common_keys)
+
+
+def check_chains(obs_chains, inp_r_chain, inp_l_chains):
```

Comment:
if the elements will not/should not be repeated, best to use `set` instead of list

Reply:
I agree, but the set type is not recognized by the defaults.yaml format. I would skip the conversion in all the modules that require this parameter. The logic takes care of possible duplicates already.